sktime/forecasting/var_reduce.py

```diff
@@ -0,0 +1,369 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements VARReduce, a VAR-like model combining tabularization with regression."""
+
+__author__ = ["meraldoantonio"]
+
+import numpy as np
+import pandas as pd
+
+from sktime.forecasting.base import BaseForecaster, ForecastingHorizon
+
+
+class VARReduce(BaseForecaster):
+    """
+    A flexible VAR-like forecaster that combines tabularization with regression.
+
+    Fitting proceeds in two steps:
+    1. Tabularization: For each time step and each time series within the
+       multivariate input data, lagged values `X` are generated.
+       These, along with the training data themselves,
+       are reshaped into a table to facilitate regression.
+
+    2. Regression: For each series `y` in the input data, a separate regressor is
+       trained using `y` and the common predictors `X`.
+
+    For forecasting, recent observations in the input data are converted to lagged
+    predictors and passed to the trained regressors to obtain forecasted values.
+
+    By default, `LinearRegression` is used, yielding results equivalent to
+    a traditional VAR model. Alternatively, any scikit-learn compatible regressor can
+    be used to introduce regularization and/or non-linearity.
+
+    For example:
+    - VARReduce(regressor = Ridge()) is equivalent to VAR with L2 regularization;
```

Comment:
minor remark: does this display properly, or do we need to add spaces around the bullet points?

Reply:
Yes I added a tab before the dash

Reply:
(Addressed and resolved)